internal/provider/resource_aci_endpoint_security_group_test.go

```diff
@@ -370,25 +367,29 @@ resource "aci_endpoint_security_group" "test" {
 `
 
 const testConfigFvESgAllDependencyWithFvAp = testConfigFvApMinDependencyWithFvTenant + `
+data "aci_system" "version" {
+  pod_id    = "1"
+  system_id = "1"
+}
+
 resource "aci_endpoint_security_group" "test" {
   parent_dn = aci_application_profile.test.id
   name = "test_name"
-  admin_state = "no"
```

Comment:
Why are we removing these?

Reply:
Introduced version_mismatch in default which takes care of this.